### PR TITLE
[GS-332] dependency updates to support courier upgrades

### PR DESCRIPTION
**Pull request by typeoneerror**
[Liberator](https://github.com/PrecisionNutrition/liberator/pull/232) <- [es-student](https://github.com/PrecisionNutrition/es-student/pull/1455) <- [courier](https://github.com/PrecisionNutrition/courier/pull/118) <- [fullscreen-card-renderer](https://github.com/PrecisionNutrition/fullscreen-card-renderer/pull/245) <- date-picker

Read the commits. 

Mostly deps updates, one test fix.

Next up:

- [ ] update date-picker in FCR
- [ ] update FCR in courier
- [ ] update courier and FCR in es-student

### CODE REVIEW

**Comment by typeoneerror** [comment]
@jherdman bit of a rabbit hole here, but finally got this linted and passing. I think the canary is safe to ignore. looks like the same fails as FCR.

**Comment by jherdman** [comment]
Works for me. I'm going to merge this. I'm going to merge a few more PRs here before you do a release (Prettier, Github Actions) today.

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "ember-cli-htmlbars": "^4.2.3",
     "ember-composable-helpers": "^4.4.1",
     "ember-power-datepicker": "^0.8.1",
-    "ember-truth-helpers": "^3.0.0"
+    "ember-truth-helpers": "^2.1.0"
```
Comment (jherdman) [inline package.json:36]: Can you discuss this change?

**Review by jherdman — CHANGES_REQUESTED** [review]
One note before I merge

**Pull request merged**